pytest_django/fixtures.py

```diff
@@ -291,11 +291,12 @@ def admin_user(db, django_user_model, django_username_field):
     try:
         user = UserModel._default_manager.get(**{username_field: username})
     except UserModel.DoesNotExist:
-        extra_fields = {}
-        if username_field not in ("username", "email"):
-            extra_fields[username_field] = "admin"
         user = UserModel._default_manager.create_superuser(
-            username, "admin@example.com", "password", **extra_fields
+            **{
+                username_field: username,
+                "email": "admin@example.com",
+                "password": "password",
```

Comment:
In case `username_field` is email, it should override the `email` below, so should come after it (even though the values are the same, but just for good measure).

```suggestion
                "email": "admin@example.com",
                "password": "password",
                username_field: username,
```

Reply:
Thanks for taking the time to review! I will add the requested changes and push again.